packages/webcert/src/store/certificate/certificateReducer.ts

```diff
@@ -269,9 +271,13 @@ const certificateReducer = createReducer(getInitialState(), (builder) =>
         return
       }
 
-      state.certificate.data[action.payload].disabled = true
-      ;(state.certificate.data[action.payload].value as ValueCode).id = ''
-      ;(state.certificate.data[action.payload].value as ValueCode).code = ''
+      const question = state.certificate.data[action.payload]
+
+      question.disabled = true
+      if (question?.value?.code) {
+        ;(question.value as ValueCode).id = ''
+        ;(question.value as ValueCode).code = ''
```

Comment:
What does the ; mean? Can you elaborate?

Reply:
Good question, for some reason prettier want's this, I guess it's having issues with parsing instances where type assignment and variable assignment are done simultaneous. I am not entirely sure.

![2022-10-07 10_56_45-● certificateReducer ts - frontend - Visual Studio Code](https://user-images.githubusercontent.com/28404/194516031-d8a7188b-d4ee-454e-a275-0134ffa2429d.png)